patches.json

```diff
@@ -138,6 +138,9 @@
     "Add the possibility to install Magento without admin creation" : {
       "2.1.4 - 2.2.1": "MAGECLOUD-2573__installation_without_admin_creation__2.1.4.patch",
       "2.2.2 - 2.2.7": "MAGECLOUD-2573__installation_without_admin_creation__2.2.2.patch"
+    },
+    "Fix monolog Slack Handler bug for magento 2.1.x": {
+      "~2.1.4": "MAGECLOUD-2793__fix_monolog_slack_handler_2.1.x.patch"
```

Comment:
You should depend on the `monolog/monolog` package rather Magento

Reply:
As I remember dependencies on modules were removed in this task https://magento2.atlassian.net/browse/MAGECLOUD-1648

```
To do:
..
Change patch constraints so they're based on Magento version and not individual module versions.
```

Reply:
Correct. Since Magento versions have hard-coded version of libraries, we depend on core version for the mapping to be more straightforward and easy to read.

Reply:
Version of lib is not likely to change. Just confirmed with patch team:
`correct, we do only security fixes for 2.1.x, so external library can be updated only in cases when the current version is no longer supported or missing some security fixes`

We should be safe to use core version here but can use module version as an exception here, since constraint is still open and there is some risk that dependency will break if eventually monolog requires a security fix.

Reply:
All Magento 2.1.x versions are using "monolog/monolog": "1.16.0". So, I applied this patch to "~2.1.4".
Since Magento 2.2.0 constraint was changed to "monolog/monolog": "^1.17".

Reply:
I will change dependency on "monolog/monolog".